### ISSUE TITLE
自动删除标签失效

### ISSUE THREAD

**Issue by MODOUser** [zoushucai/zotero-journalabbr#23]
zotero版本：7.0.0-beta.53+969031a37(64-bit)
插件版本：0.7.10

对一篇无标签的期刊采用iso缩写和内置缩写能够正常打相应的标签，但是先使用iso或者内置再使用另一种并不会删除对应标签，最后显示如图的内容。自定义的我没试，因为还没有自定义csv。
此外打regex标签也不会自动删除abbr和abbr_iso标签
![image](https://github.com/zoushucai/zotero-journalabbr/assets/51368808/6acb6564-6942-4fd0-b59b-2f167c7f38c8)

**Comment by MODOUser** [comment]
还有就是采用某一种缩写更新刊名简称的时候可以顺便更新abbr字段吗，目前好像只有replace功能可以更新abbr，其他的方式建议也更新abbr，因为期刊和会议不一样，会议似乎没有刊名简称字段，所以我正则表达式一般会把conferenceName设置为会议的简称，最后统一更新到abbr，这样只需要常驻一个abbr字段就可以查看期刊和会议的简称了

**Comment by zoushucai** [comment]
寒假的时候改改

**Comment by zoushucai** [comment]
在新的版本中,已修复自动删除标签的问题

**Comment by zoushucai** [comment]
> 还有就是采用某一种缩写更新刊名简称的时候可以顺便更新abbr字段吗，目前好像只有replace功能可以更新abbr，其他的方式建议也更新abbr，因为期刊和会议不一样，会议似乎没有刊名简称字段，所以我正则表达式一般会把conferenceName设置为会议的简称，最后统一更新到abbr，这样只需要常驻一个abbr字段就可以查看期刊和会议的简称了
是的, 这是其中一种方式, 还有一种方式, 直接点击 `abbrall` 也可以更新到 abbr 字段 (由于内置了 3M 的缩写数据,首次运行可能会有点卡)

**Comment by zoushucai** [comment]
> 还有就是采用某一种缩写更新刊名简称的时候可以顺便更新abbr字段吗，目前好像只有replace功能可以更新abbr，其他的方式建议也更新abbr，因为期刊和会议不一样，会议似乎没有刊名简称字段，所以我正则表达式一般会把conferenceName设置为会议的简称，最后统一更新到abbr，这样只需要常驻一个abbr字段就可以查看期刊和会议的简称了

是的, 这是其中一种方式, 还有一种方式, 直接点击 `abbrall` 也可以更新到 abbr 字段 (由于内置了 3M 的缩写数据,首次运行可能会有点卡)
